mbed/.gitignore

```diff
@@ -1,12 +1,12 @@
-.build
-.mbed
-cmake_build
-projectfiles
-*.py*
-mbed-os
-mbed-os-ble-utils
-BUILD
-GPATH
-GRTAGS
-GTAGS
+.build
```

Comment:
why is github showing a diff here? funny line endings?

Reply:
Yes, the old file had CRLF as I copied it from Windows